openapi/golem-service.yaml

```diff
@@ -2848,19 +2854,19 @@ components:
         entries:
           type: array
           items:
-            $ref: '#/components/schemas/PublicOplogEntry'
+            $ref: '#/components/schemas/PublicOplogEntryWithIndex'
         next:
           $ref: '#/components/schemas/OplogCursor'
-        first_index_in_chunk:
+        firstIndexInChunk:
```

Comment:
is this okay to change?

Reply:
I think yes because this is a 1.1 feature (just added in the previous oplog enumeration PR). The serde/oai attributes were missing that are making it came case, which we have for other types that are used in the REST API.